e2e/e2e_test.go

```diff
@@ -307,6 +335,32 @@ var _ = Describe("kubectl accurate", func() {
 		Expect(tokenName2).NotTo(Equal(tokenName))
 	})
 
+	It("should propagate ServiceAccount w/o secrets field (Kubernetes 1.24 or higher)", func() {
+		if k8sMinorVersion < 24 {
+			Skip("this test case does not work")
+		}
+
+		// From Kubernetes 1.24, the auto-generation of secret-based service account tokens has been disabled by default.
+		// So the secrets field in the ServiceAccount is not updated. But when upgrading Kubernetes from 1.23 or lower,
+		// some ServiceAccounts that have been created before the upgrade might have the secrets field.
+		// In this case, accurate should not copy the field.
+		kubectlSafe(serviceAccountWithDummySecretsYAML, "apply", "-f", "-")
+		Consistently(func() error {
```

Comment:
The creation of `sn1/test` should be waited with `Eventually()`.
This `Consistently()` can fail if the creation of `sn1/test` ServiceAccount is delayed.
As far as I know `kubectl apply` above is synchronous only for `subroot1/test`, not for `sn1/test`.

Once `sn1/test` is created, it would be enough to check the contents of the ServiceAccount just once because we know that the ServiceAccount is created atomically.
To test the stability of `sn1/test` seems excess for me.

You can check the contents even inside `Eventually()` as done in the previous `It()`.

Reply:
> This Consistently() can fail if the creation of sn1/test ServiceAccount is delayed.

Thank you. That's true. If the test machine has low spec, this test may be fail.
I'll use `Eventually()`.